src/test/scala/com/scalableminds/fossildb/FossilDBSuite.scala

```diff
@@ -30,7 +30,7 @@ class FossilDBSuite extends FlatSpec with BeforeAndAfterEach with TestHelpers wi
   private val testData3 = ByteString.copyFromUtf8("testData3")
 
   private val aKey = "aKey"
-  private val anotherKey = "anotherKey"
+  private val aNotherKey = "aNotherKey"
```

Comment:
this change to make the tests more intuitive (keys are alphabetically sorted now)